frontend/storage/common.ts

```diff
@@ -1,21 +1,22 @@
 // eslint-disable-next-line no-unused-vars
-import {GameSpoilers} from './models/GameSpoilers';
+import { Game } from './models/Game';
 export const SET_BREADCRUMB = 'SET_BREADCRUMB';
-export const SET_GAME_SPOILERS = 'SET_GAME_SPOILERS';
+export const SET_GAME_VISIBILITY = 'SET_GAME_VISIBILITY';
 
 export interface AppStore {
 	headerBreadcrumb: string;
-	gameSpoilers: Map<number, GameSpoilers>;
+	gamesVisibility: Map<number, boolean>;
+	games: Map<number, Game>;
 }
 export interface SetBreadcrumbAction{
 	type: typeof SET_BREADCRUMB;
 	headerBreadcrumb: string;
 }
-export interface setGameSpoilersAction{
-	type: typeof SET_GAME_SPOILERS;
+export interface SetGameVisibilityAction{
+	type: typeof SET_GAME_VISIBILITY;
 	payload: {
```

Comment:
In `SetBreadcrumbsAction` you have the payload directly in the object, while in this action you group it under `payload`. Chose one solution and stick to it - I personally prefer just having them directly in the action, because I don't feel the payload is beneficial in any scenario I'd use.

There are some arguments for keeping it in the dedicated `payload` though, like making it easier to extract the payload from the action, but I can't think of a usecase at the moment.